hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/RowCounter.java

```diff
@@ -64,13 +64,15 @@
 
   private final static String OPT_START_TIME = "starttime";
   private final static String OPT_END_TIME = "endtime";
+  private final static String OPT_IS_CACHE_BLOCKS = "isCacheBlocks";
   private final static String OPT_RANGE = "range";
   private final static String OPT_EXPECTED_COUNT = "expectedCount";
 
   private String tableName;
   private List<MultiRowRangeFilter.RowRange> rowRangeList;
   private long startTime;
   private long endTime;
+  private boolean isCacheBlocks;
```

Comment:
isCacheBlocks is name of the method that looks at the cacheBlock data member content, not the name of the data member... so suggest s/isCacheBlocks/cacheBlocks/

Reply:
Okay, I'll fix it right away, as well as the changes I missed to submit last time